worker/build.rs

```diff
@@ -78,19 +80,26 @@ fn main() {
     #[cfg(target_os = "windows")]
     {
         if !std::path::Path::new("worker/out/msys/bin/make.exe").exists() {
-            let python = if Command::new("where")
-                .arg("python3.exe")
+            let python = if env::var("PYTHON").is_ok() {
+                env::var("PYTHON").unwrap()
+            } else if Command::new("where")
+                .arg("python3")
                 .status()
                 .expect("Failed to start")
                 .success()
             {
-                "python3"
+                "python3".to_string()
             } else {
-                "python"
+                "python".to_string()
             };
 
+            let worker_abs_path = env::current_dir().unwrap();
+            let dir = Path::new(&worker_abs_path).join("out/msys");
```

Comment:
I'm pretty sure this will also work:
```suggestion
            let dir = worker_abs_path.join("out/msys");
```

Reply:
What is `worker_abs_path`? Is it a simple string? or is it a `Path` instance? If it's a string I prefer to use `path` utils to properly join them given `/` or `\\` syntax to separate folders in Windows and Unix.

Reply:
It is faster to check docs than asking me to reply with a link: https://doc.rust-lang.org/stable/std/env/fn.current_dir.html
Also IDE should ideally tell you that :confused: